vmm/src/api/mod.rs

```diff
@@ -196,6 +196,7 @@ pub struct VmSendMigrationData {
     /// URL to migrate the VM to
     pub destination_url: String,
     /// Send memory across socket without copying
+    #[serde(default)]
```

Comment:
Here is a slightly better commit message

```
vmm: api: Make for VmSendDataMigration.local optional

Fixes: #3756
```

Reply:
done